tests/unit/test_global_config.py

```diff
@@ -35,6 +35,26 @@ def test_global_config_parse(filename):
     assert global_config.has_dashboard() is True
 
 
+def test_global_config_from_app_manifest(app_manifest_correct):
+    expected_global_config = global_config_lib.GlobalConfig(
+        {
+            "meta": {
+                "name": "Splunk_TA_UCCExample",
+                "restRoot": "Splunk_TA_UCCExample",
+                "displayName": "Splunk Add-on for UCC Example",
+                "version": "7.0.1",
+            }
+        },
+        False,
+    )
+
+    global_config = global_config_lib.GlobalConfig.from_app_manifest(
+        app_manifest_correct
+    )
+
+    assert expected_global_config == global_config
```

Comment:
I think instead of adding comparison method in source code only for tests, we could have a helper method that does the same job as we do in `__eq__()` method. Thoughts?

Reply:
Yeah, I could do that but I like keeping such logic in the class itself.

If I would be writing a test to compare 2 objects - I assume
this object has eq method implemented.